project_stock_product_set/tests/test_project_stock_product_set.py

```diff
@@ -21,7 +22,9 @@ def setUpClass(cls):
             }
         )
 
+    @users("basic-user")
```

Comment:
Do the test also with admin. This way, if both tests fail in a new version, we know that it's not permission issue.